students/david_baylor/lesson4/Mail_Room_part2.py

```diff
@@ -8,41 +8,49 @@
 Automates writing an email to thank people for their donations.
 """
 
+import sys
+from operator import itemgetter
 
 def send_thank_you(data):
     name = input("Enter a full name or type 'list' to vew current list: ")
     if name.upper() == "LIST":
-        for i in range(len(data)):
-            print(data[i]["name"].capitalize())
+        for i in data:
+            print(i.capitalize())
         return data
     else:
-        for i in range(len(data)):
-            if name.upper() == data[i]["name"].upper(): 
+        for i in data:
+            if name.upper() == i.upper(): 
```

Comment:
This should be updated to utilize efficient dictionary lookups (hint: you should use `get()` method on the dictionary here.